core/doc/snippets/uServices-modulecontext/main.cpp

```diff
@@ -16,9 +16,7 @@ void RetrieveModuleContext()
 //! [GetModuleContext]
 
 //! [InitializeModule]
```

Comment:
You may want to remove the comments denoting the block since the code block does not exist any more. Initialization happens elsewhere ...